library/common/config_template.cc

```diff
@@ -70,6 +70,12 @@ const char* config_template = R"(
         keepalive_interval: 10
         keepalive_probes: 1
         keepalive_time: 5
+    circuit_breakers: &circuit_breakers_settings
+      thresholds:
+        - priority: DEFAULT
+          retry_budget:
+            budget_percent:
+              value: 50
```

Comment:
Curious, why not set this to 100? Also looks like this defaults to 20%, how does that work with the 3 mentioned in the description? https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/cluster/circuit_breaker.proto